docs/intro/ink-vs-solidity.md

```diff
@@ -905,14 +905,13 @@ mycontract = { path = "mycontract/", default-features = false, features = ["ink-
 ## unit testing (off-chain)
 
 - Unit tests are an integral part of smart-contract development and ensuring your code works off-chain before testing on-chain.
-- To run ink! tests, do _not_ use `cargo +nightly contract test`. Use `cargo +nightly test`. Add the `--nocapture` flag for debug prints to show. See [here](https://substrate.stackexchange.com/questions/3197/how-to-understand-which-test-failed-in-ink) for more info on why.
+- To run ink! tests, do _not_ use `cargo contract test`. Use `cargo test`. Add the `--nocapture` flag for debug prints to show. See [here](https://substrate.stackexchange.com/questions/3197/how-to-understand-which-test-failed-in-ink) for more info on why.
```

Comment:
`cargo contract test` no longer exists, so we can probably remove this part

Reply:
Thanks for the feedback, it's done.